scripts/requirements-extras.txt

```diff
@@ -22,7 +22,8 @@ Pillow
 imgtool>=1.9
 
 # used by nanopb module to generate sources from .proto files
-grpcio-tools
+grpcio-tools>=1.47.0
+protobuf>=3.20.3
```

Comment:
This will break the global installation (i.e. outside a virtual environment) on some Linux distributions (e.g. Ubuntu) because `protobuf` is "externally managed" and provided by the `python3-protobuf` package.

But, given that some distro-provided versions are incompatible, it looks like there is no other choice.

For the most part, this should be ok since our Getting Started Guide now recommends the virtual environment-based installation by default.